pkg/iostreams/console_windows.go

```diff
@@ -0,0 +1,21 @@
+// +build windows
+
+package iostreams
+
+import (
+	"os"
+
+	"golang.org/x/sys/windows"
+)
+
+func (s *IOStreams) EnableVirtualTerminalProcessing() {
```

Comment:
I am not familiar with windows console modes but do we need to disable/reset  `ENABLE_VIRTUAL_TERMINAL_PROCESSING` to its original value before `gh` quits?

Reply:
my understanding is that it only has effect while `gh` is running. i can try and get clarity on that

Reply:
I've stared at this more and it seems to apply to a specific buffer for a given file handle which means nothing would persist once gh quits.